ansible-ipi-install/roles/node-prep/tasks/10_validation.yml

```diff
@@ -129,6 +129,14 @@
   - always
   - validation
 
+- name: Confirm whether or not internet connectivity on provisioner host
+  uri:
+    url: https://www.redhat.com
+    status_code: [-1,200]
+    timeout: 1
+  register: the_url
```

Comment:
so existing lines 148-155 already have this exact code but the variable is labeled "check_url". We could likely leverage that. The only issue I worry about is since the variable would be declared in the node-prep role, I'm unsure by the time it reaches the installer role if it would still be usable.

Reply:
When it was not here, and using the 'tags cache', it was failing as the variable was not defined, moving this from the other module to this one, allowed it to pass, should we reuse check_url and the_url everywhere we check connectivity?

Reply:
If we tag that task with cache and then include the cache tag in the main.yml under 10_validation does it work? Versus re-including the same task somewhere else.